src/code/utilities/firebase-auth.js

```diff
@@ -0,0 +1,134 @@
+/* global firebase */
+import urlParams, {updateUrlParameter} from "./url-params";
+import jwt from 'jsonwebtoken';
+
+export const initFirebase = new Promise(function (resolve, reject) {
+  // Initialize Firebase
+  const config = {
+    apiKey: "AIzaSyCr8UbzmHqWVuOIQrU2_1_CIIwT-GphnYo",
+    authDomain: "gvstaging.firebaseapp.com",
+    databaseURL: "https://gvstaging.firebaseio.com",
+    projectId: "gvstaging",
+    storageBucket: "",
+    messagingSenderId: "574673678327"
+  };
+  // TODO: live fb app name needs checking
+  // const configLive = {
+  //   apiKey: "AIzaSyCQyZqErr-WsvaZzATcmOgxxv1wcrNQXIo",
+  //   authDomain: "gvdemo-6f015.firebaseapp.com",
+  //   databaseURL: "https://gvdemo-6f015.firebaseio.com",
+  //   projectId: "gvdemo-6f015",
+  //   storageBucket: "",
+  //   messagingSenderId: "574673678327"
+  // };
```

Comment:
This is going to need to be changed before pulling in, but we'd probably like to always use the staging Firebase DB automatically every time we deploy to staging, right?

Could we keep both versions of the config, and switch between them if the string `/branch/tutorial/` is in the url path? (Easier than a url parameter that would need to play nicely with the parameters being set from the portal)